indico/web/forms/widgets.py

```diff
@@ -159,3 +159,29 @@ def __call__(self, field, **kwargs):
 
         options.update(kwargs.pop('options', {}))
         return super(SelectizeWidget, self).__call__(field, options=options)
+
+
+class StaticSelectizeWidget(JinjaWidget):
```

Comment:
I think we could probably unify both `SelectizeWidget`s, especially considering they're used so few times. We could allow e.g. a parameter with the AJAX call to be specified, or a predefined list of options instead.